libgalois/include/katana/Properties.h

```diff
@@ -507,17 +507,17 @@ struct StructProperty
 
 
 template <typename T>
-class VectorRef {
+class ArrayRef {
   public:
     using value_type = T;
     using iterator = value_type*;
     using const_iterator = const value_type*;
 
-    VectorRef(T* ptr=NULL, size_t size=1): data_(ptr), size_(size){}
+    ArrayRef(T* ptr=NULL, size_t size=1): data_(ptr), size_(size){}
```

Comment:
Can we remove these default arguments? NULL, 1 represents an invalid state for this class